### DISCUSSION TITLE
Resources to follow tutorials

### DISCUSSION THREAD

**Discussion by jesswhitts** [aertslab/scenicplus#210] [Q&A] [answered]
Hi, 

I am would like to try SCENIC+ by first following the melanoma cell line tutorial (as my data is not multiome). Are the files used for this tutorial available to download somewhere?

I know the raw files have been deposited, but I can't find the demuxlet files.

Many thanks,
Jess

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @jesswhitts 

If you send me your mail adres I'll send you a download link.

Best,

Seppe

**Reply by jesswhitts** [reply]
jessica.whittle@cruk.manchester.ac.uk

Many thanks!

**Comment by ytrink** [comment]
Hi @SeppeDeWinter  would it be possible to get the download link as well?
trinkya@biu.ac.il
Thank you,
Yaron

**Comment by mz5252** [comment]
Hi @SeppeDeWinter ,

I am reproducing the melanoma cell-line analysis using GSE210745. Could you please share the following demuxlet outputs or a download link?

four_lines_demuxlet.best
seven_lines_demuxlet.best

A barcode-to-cell-line assignment table would also work. Could you also include the mapping between the demuxlet donor/sample IDs and the MM cell-line names?

mz52@illinois.edu
Thank you!
Mianchen

**Comment by SeppeDeWinter** [comment]
Hi @mz5252

Please find those files below.

[demuxlet.zip](https://github.com/user-attachments/files/30035658/demuxlet.zip)

best,

Seppe
